src/CSSmodule/bookitem.css

```diff
@@ -0,0 +1,144 @@
+.bookList {
+  margin: 2rem 3rem;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.bookSection {
+  display: flex;
+  width: inherit;
+  justify-content: space-between;
+  padding: 10px;
+  background-color: #fff;
+}
+
+.userContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.detail {
+  display: flex;
+  flex-direction: column;
+}
+
+.titles {
```

Comment:
- [ ] Good job working on the styling, but please kindly ensure that the circular progress is the same size with that of the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/208231574-a1e112aa-89c2-41d2-b211-d12663d6c715.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/208231560-e3aa4efc-91d0-4928-bd33-ba8a1913d34c.png)